src/Blt/Plugin/Commands/SimpleSamlPhpCommand.php

```diff
@@ -70,6 +70,20 @@ public function initializeSimpleSamlPhp() {
     $this->outputCompleteSetupInstructions();
   }
 
+  /**
+   * Adds simplesamlphp_auth as a dependency.
+   *
+   * @throws \Acquia\Blt\Robo\Exceptions\BltException
+   */
+  protected function requireModule() {
```

Comment:
This function isn't necessary because `drupal/simplesamlphp_auth` is already a dependency in composer.json. That was a large part of the motivation to make functionality with Composer dependencies into plugins, because running Composer programmatically is really asking for trouble.

Reply:
I noticed that.

The first error I got was

```
Fatal error: Uncaught Error: Call to undefined method Acquia\BltSimplesamlphp\Blt\Plugin\Commands\SimpleSamlPhpCommand::requireModule() in /app/vendor/acquia/blt-simplesamlphp/src/Blt/Plugin/Commands/SimpleSamlPhpCommand.php:65
```

So I started by restoring the missing function.

I will update the PR to remove the call to `requireModule()` instead of restoring the function.

Reply:
Done: I updated the PR and tested on my current project.

Reply:
Ah yeah, that makes sense, it should have been removed.